staging/src/kubevirt.io/client-go/go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	github.com/coreos/prometheus-operator v0.38.0
 	github.com/go-kit/kit v0.9.0
 	github.com/go-openapi/spec v0.19.3
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
In this file we still have go 1.12.

Reply:
Ouch, I was searching for 1.16, and it slip out to me. Thanks